### PR TITLE
Docs for CLI features in 1.3

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

Adds configurable options and commands for new buckets and databases CLI deployment commands.

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: app/views/docs/command-line-ci.phtml [hunk 1]
```diff
@@ -35,8 +35,8 @@ The Appwrite CLI can be used in a non-interactive and headless manner, without s
 </div>
 
 
-<p>You can deploy collections and teams non-interactively in a similar way by using the <code>--yes</code> option.</p>
-<p>Deploy all collections:</p>
+<p>You can deploy databases, collections, and teams non-interactively in a similar way by using the <code>--all</code> and <code>--yes</code> option.</p>
```
Comment (Meldiron) [inline app/views/docs/command-line-ci.phtml:38]:
```suggestion
<p>You can deploy databases, collections, buckets and teams non-interactively in a similar way by using the <code>--all</code> and <code>--yes</code> option.</p>
```

File: app/views/docs/command-line-ci.phtml [hunk 2]
```diff
@@ -45,3 +45,9 @@ The Appwrite CLI can be used in a non-interactive and headless manner, without s
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="CLI">
     <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite deploy team --all --yes</code></pre>
 </div>
+
+<p>Deploy all buckets:</p>
```
Comment (Meldiron) [inline app/views/docs/command-line-ci.phtml:49]: Let's add one for teams too.

File: app/views/docs/command-line-deployment.phtml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <p>
-The Apprite CLI allows you to create and deploy collections and functions to your Appwrite project from a configuration file. This is especially helpful if you're trying to track functions and collection schemas using version control, or if you're promoting changes from development to production environments.
+The Apprite CLI allows you to create and deploy databases, collections, and functions to your Appwrite project from a configuration file. This is especially helpful if you're trying to track functions and collection schemas using version control, or if you're promoting changes from development to production environments.
```
Comment (Meldiron) [inline app/views/docs/command-line-deployment.phtml:2]:
```suggestion
The Apprite CLI allows you to create and deploy databases, collections, buckets, teams and functions to your Appwrite project from a configuration file. This is especially helpful if you're trying to track functions and collection schemas using version control, or if you're promoting changes from development to production environments.
```

File: app/views/docs/command-line-deployment.phtml [hunk 4]
```diff
@@ -45,14 +45,22 @@ The Apprite CLI allows you to create and deploy collections and functions to you
...
+<h2><a href="/docs/command-line-deployment#deployCollections" id="deployCollections">Deploying Databases and Collections</a></h2>
 
-<p>The Appwrite CLI also helps you migrate your project's collections from a development server to a production server. You can deploy all the collections in your <a href="/docs/command-line-deployment#appwriteJSON">appwrite.json</a> file using:</p>
+<p>The Appwrite CLI also helps you migrate your project's databases and collections from a development server to a production server. You can deploy all the databases and collections in your <a href="/docs/command-line-deployment#appwriteJSON">appwrite.json</a> file using:</p>
 
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="CLI">
     <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite deploy collections</code></pre>
 </div>
 
+<h2><a href="/docs/command-line-deployment#deployBucket" id="deployBucket">Deploying Storage Buckets</a></h2>
```
Comment (Meldiron) [inline app/views/docs/command-line-deployment.phtml:56]: Let's have one more for teams, and one for functions

File: app/views/docs/command-line-deployment.phtml [hunk 5]
```diff
@@ -96,6 +109,11 @@ The Apprite CLI allows you to create and deploy collections and functions to you
             <td>array of <a href="/docs/command-line-deployment#teams-options">teams</a></td>
             <td>Configuration of teams in your project.</td>
         </tr>
+        <tr>
+            <td><code>buckets</code></td>
```
Comment (Meldiron) [inline app/views/docs/command-line-deployment.phtml:113]: Teams missing

**Review by Meldiron — COMMENTED** [review]
Let's please double check that we always mention that CLI has those features: databases & collections, buckets, functions, teams

Ooor let's maybe avoid being specific if not necessary. We will be adding more features in future. Some teams docs are already missing, which is feature that is already released.

File: app/views/docs/command-line-deployment.phtml (hunk 5 above)
Comment (gewenyu99) [reply app/views/docs/command-line-deployment.phtml:113]: Look up ^^ !

File: app/views/docs/command-line-deployment.phtml (hunk 4 above)
Comment (gewenyu99) [reply app/views/docs/command-line-deployment.phtml:56]: Done

**Review by Meldiron — APPROVED** [review]

**Review by abnegate — APPROVED** [review]

**Pull request merged**